modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/macros/FDSCustomView.macro

```diff
@@ -43,6 +43,24 @@ definition {
 			locator1 = "Button#GENERAL_BUTTON_MODAL_FOOTER");
 	}
 
+	macro deleteView {
+		Click(locator1 = "FrontendDataSet#SELECT_OPTION_HEADER");
+
+		Click(
+			key_itemName = "Delete View",
+			locator1 = "FrontendDataSet#SELECT_OPTION_CUSTOM_VIEW");
+
+		AssertTextEquals(
```

Comment:
Do not use assertions in macros that are not meant to be in `Then` step assertions.